core/jobs/batch_jobs/manual_voice_artist_name_job.py

```diff
@@ -78,37 +77,6 @@ def is_exploration_curated(exploration_id: str) -> Optional[bool]:
                 ' for exploration ID %s.' % exploration_id)
             return False
 
-    @classmethod
-    def is_voiceover_changes_made(
```

Comment:
This is no longer required, as we will now get the list of voiceover filenames added in a specific version using the two successive versions of the exploration. And if the list of filenames is empty, we assume that no voiceovers were added in that version.